content/blog/2021-07-16-july-21-dvc-heartbeat.md

```diff
@@ -0,0 +1,247 @@
+﻿---
+title: July '21 Heartbeat
+date: 2021-07-16
+description: |
+  Monthly updates are here! Great new tutorials from the Community, uptick in 
+  jobs requiring DVC, awesome Community discussion on experiments at our June
+  Meetup and a cipher. Can you figure it out?
+descriptionLong: |
+  This month you will find:
+  📈 DVC + Streamlit = ❤️,
+  🇯🇵 DVC in Japanese,
+  📖 A new Udacity Course that includes DVC,
+  🧑🏽‍💻 More and more jobs requiring DVC
+  🧪 June Meetup on Experiments,
+  🚀 New team member, a secret code and more!
+picture: 2021-07-16/july21cover.png
+author: jeny_defigueiredo
+commentsUrl: https://discuss.dvc.org/t/july-heartbeat/825
+tags:
+  - Heartbeat
+  - DVC
+  - CML
+  - Streamlit
+  - Udacity
+---
+
+# Welcome to Summer!
+
+![It's summer!](https://media.giphy.com/media/WuY9yfI89DbNu/giphy.gif)
+
+# From the Community
+
+<span style="color:purple">A</span>s usual we have a ton of goodness from the
+Community! Let's jump in!
+
+## Antoine Toubhans Post Combining Streamlit and DVC!
+
+[Antoine Toubhans](https://www.linkedin.com/in/antoine-toubhans-92262119/) of
+[Sicara](https://www.sicara.fr/) wrote a fantastic and detailed tutorial
+bringing together the best of DVC and integrating it with Streamlit to provide a
+customizable UI. The tutorial goes through the steps of setting up a pipeline,
+spltting a dataset, training and evaluating a model, tracking changes to data
+and model, dvc <span style="color:purple">m</span>etrics and plots and then
+bridging the gap in visualizations using Streamlit. You won't want to miss this
+one!
+
+<external-link
+href="https://www.sicara.ai/blog/dvc-streamlit-webui-ml"
+title="How to Build Customizable Web UI for Machine Learning with Streamlit and DVC"
+description="Antoine Toubhans shares a detailed tutorial on using DVC with Streamlit to provide a customizable UI"
+link="https://www.sicara.ai/"
+image="/uploads/images/2021-07-16/streamlit.png"/>
+
+https://media.giphy.com/media/l0MYGtVSzAqKfkcEg/giphy.gif
```

Comment:
is it better to take an image for the post (may add a title about it's source and a link in the subtitle)

Reply:
Ok.  I will do.  I was on the fence about this gif, cuz it's so big anyway.